src/api/database/changes.rst

```diff
@@ -95,7 +95,8 @@
                            - :mimetype:`text/plain; charset=utf-8`
     :>header ETag: Response hash is changes feed is `normal`
     :>header Transfer-Encoding: ``chunked``
-    :>json number last_seq: Last change sequence number
+    :>json string last_seq: Last change sequence identifier
```

Comment:
it can be any valid JSON, classifying this item as a string is the same mistake as classifying it as number.